packages/code/src/Code.tsx

```diff
@@ -140,8 +282,34 @@ function Code({
   chromeTitle = '',
   ...rest
 }: CodeProps) {
+  const scrollableMultiLine = useRef<HTMLPreElement>(null);
+  const scrollableSingleLine = useRef<HTMLDivElement>(null);
+  const [state, dispatch] = useReducer(scrollReducer, initialState);
+  useLayoutEffect(() => {
+    if (!state.isLoaded) {
+      dispatch({ type: 'load' });
+      if (multiline) {
+        if (
+          scrollableMultiLine.current !== null &&
```

Comment:
I might recommend destructuring these to make the code overall more readable:

```js
const { current: multiLineEl } = scrollableMultiLine

if (
  multiLineEl != null &&
  multiLineEl.scrollWidth > multiLineEl.clientWidth
) { ... }
```

Also, did you mean to do `scrollableMultiLine.current != null`?